libs/design/src/atoms/form/input/input/input.component.scss

```diff
@@ -9,9 +9,18 @@
     font-size: 1rem;
```

Comment:
Styles should be what I changed in this closed PR (https://github.com/graycoreio/daffodil/pull/195/files#diff-3b23dd3db08e6d8efeceed687f95890e)